app/subscriber/src/features/home/Home.tsx

```diff
@@ -54,7 +54,15 @@ export const Home: React.FC = () => {
   );
 
   const homePage = React.useMemo(() => {
-    return getFilterActions(actions)[ActionName.Homepage];
+    const featured = actions.find((a) => a.name === 'Featured Story');
```

Comment:
Not sure of what the best way to do this would be other than hard coding / making another magic enum

Reply:
As the names are managed by admins on the site

Reply:
This is what Settings are for.  We created them for dynamic constants.

Reply:
I may be over complicating this, but in the admin page for `Actions` the admin updates the name of said actions but the `action.name` should really be paired up with `Settings` in some way? Unless we want them to change the action name (in _Admin -> Actions_), then additionally update a variable under _Admin -> Settings_

Reply:
Will adjust

Reply:
Had a d'oh moment, but updated now @Fosol